src/wallet/walletdb.cpp

```diff
@@ -832,6 +839,9 @@ DBErrors WalletBatch::LoadWallet(CWallet* pwallet)
                     // Set tx_corrupt back to false so that the error is only printed once (per corrupt tx)
                     wss.tx_corrupt = false;
                     result = DBErrors::CORRUPT;
+                } else if (wss.descriptor_corrupt) {
+                    pwallet->WalletLogPrintf("Error: Corrupt wallet descriptor found in wallet %s. This can be fixed by removing wallet from the settings so it's not loaded at startup.\n", pwallet->GetName());
```

Comment:
This is way too complicated for a typical user.

Reply:
Are you talking about GUI users?
For them, I think that the best would be to trigger a dialog and ask if want to continue without this wallet. But, that would require a bit more work that could tackle here or in a follow-up work 

For bitcoind users, as the app cannot proceed without further interaction, shouldn't be a problem for them to remove a line in the settings file. But.. another option could be to not load/open certain wallet/s, even if they are on the settings file, by providing a startup flag "-unloadwallet=<wallet_name>" (need to check if we have an available flag for this already or will have to create one). What do you think?

Note: In both cases, I don't think that skip certain wallet from loading at startup without the user interaction (nor the user knowing about it) is any better than this (would actually say that it's worst).

Reply:
I mean both should get the error details without jumping through extra hoops

Reply:
Pushed a different text version.

At this point of the code we don't have much information aside from the "unknown wallet descriptor found" (at least for now, will work on it on a follow-up PR). We just know that a certain descriptor parse failed. Thus why added the extra information at the end, to provide the user a workaround if still want to run other wallets (just need to unload the failing one).

But.. for the sake of simplicity, removed the extra information as well. Let me know what you think.

Better to work on the GUI "unload wallet confirmation dialog" and the "unload wallet" startup flag on a different PR so this bugfix can get merged for this release.